client/views/msg_condensed.tpl

```diff
@@ -0,0 +1,7 @@
+<div class="msg {{type}} closed" data-time="{{time}}">
+	<span class="time hide-text">{{tz time}}</span>
+	<span class="from"><span class="expander"></span></span>
```

Comment:
>  Anyone with strong arguments against having this on the right of the text, to make it consistent with preview toggles?

I against it, because it will a problem later, in #1047: there are clickable nicknames, so the safest way to expand it is to click on toggle icon. And here is a problem: toggle icon is small enough to miss it. Current way allow to expand condensing by clicking on a huge area, which is a big help:

![2017-07-31_08-40-03](https://user-images.githubusercontent.com/3627488/28764196-7e9ae1d4-75cc-11e7-8940-12be4be232f1.png)